client/src/pages/login/Login.tsx

```diff
@@ -28,6 +29,7 @@ const Login: FC = () => {
     console.log(submit)
     setSuccess(submit.message);
     setSubmitting(false);
+    // what's that navigate(0)?
```

Comment:
navigate(0) is doing a page refresh which rerenders the navbar. i couldnt figure out any other way to rerender the navbar